core/tests/protractor/subscriptions.js

```diff
@@ -86,10 +82,10 @@ describe('Subscriptions functionality', function() {
     creatorDashboardPage.navigateToSubscriptionDashboard();
     expect(element.all(by.css(
       '.protractor-test-subscription-name')).first().getText()).toMatch(
-      'learner...');
```

Comment:
These values should stay the same.

Reply:
okay

Reply:
Should I rename the names as they were previously? Still, we have got errors.

Reply:
Wait a sec. I think these should change to the actual strings, and we should use a toBe/toEqual check. @brianrodri -- why do you advise reverting to the more lax version of the tests?